src/Nethermind/Nethermind.JsonRpc/Modules/Eth/GasPrice.cs

```diff
@@ -0,0 +1,176 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using Jint.Native;
+using Nethermind.Blockchain.Find;
+using Nethermind.Core;
+using Nethermind.Int256;
+using Nethermind.TxPool;
+
+namespace Nethermind.JsonRpc.Modules.Eth
+{
+    public class GasPrice
+    {
+        private static UInt256? _lastPrice = null; //will this be okay if it is static?
+        private static Block? _lastHeadBlock = null;
+        private readonly IBlockFinder _blockFinder;
+        public GasPrice(IBlockFinder blockFinder)
+        {
+            _blockFinder = blockFinder;
+        }
+        private bool eth_addTxFromBlockToSet(Block block, ref SortedSet<UInt256> sortedSet, UInt256 finalPrice,
```

Comment:
If you look at the module, you will notice convention where eth_[something] names are only reserved for the methods related to the public JSON RPC endpoints. All other methods always follow standard naming from C#.
Try to look at the components that you write in and understand their conventions and logic.